### PR TITLE
🧹 add a few more details to the ecs container assets discovered

### PR DESCRIPTION
**Pull request by vjeffrey**
some of this information was missing for the scan time results of ecs containers

### CODE REVIEW

File: motor/discovery/aws/ecs.go [hunk 1]
```diff
@@ -238,11 +257,19 @@ func ecsContainerToAsset(ctx context.Context, account string, region string, c e
...
+	asset.Labels[ClusterNameLabel] = clusterName
+	asset.Labels[TaskDefinitionArnLabel] = *task.TaskDefinitionArn
 	return asset
 }
 
+const (
+	ContainerName          = "container_name"
+	ClusterNameLabel       = "cluster_name"
+	TaskDefinitionArnLabel = "task_definition_arn"
+	TaskId                 = "task_id"
```
Comment (joelddiaz) [inline motor/discovery/aws/ecs.go:269]: Didn't see where ContainerName and TaskId are used

File: motor/discovery/aws/ecs.go [hunk 2]
```diff
@@ -202,6 +206,14 @@ func ecsContainerToAsset(ctx context.Context, account string, region string, c e
 	for _, ca := range c.NetworkInterfaces {
 		containerAttachmentIds = append(containerAttachmentIds, *ca.AttachmentId)
 	}
+	taskId := ""
+	if ok := arn.IsARN(*c.TaskArn); ok {
```
Comment (joelddiaz) [inline motor/discovery/aws/ecs.go]: nit:
```suggestion
	if arn.IsARN(*c.TaskArn) {
```

**Review by joelddiaz — COMMENTED** [review]
just two nits i noticed

File: motor/discovery/aws/ecs.go (hunk 1 above)
Comment (vjeffrey) [reply motor/discovery/aws/ecs.go:269]: they will be used in the lambda/at ecs exec scan time

**Review by joelddiaz — APPROVED** [review]

**Pull request merged**